src/proposals/20250109/SparkEthereum_20250109.t.sol

```diff
@@ -0,0 +1,926 @@
+        _assertMorphoCap(Ethereum.MORPHO_VAULT_DAI_1, ptsusdemay, 0);
+
+        executeAllPayloadsAndBridges();
+
+        _assertMorphoCap(Ethereum.MORPHO_VAULT_DAI_1, usde945,    0);
+        _assertMorphoCap(Ethereum.MORPHO_VAULT_DAI_1, usde77,     0);
+        _assertMorphoCap(Ethereum.MORPHO_VAULT_DAI_1, susde945,   0);
+        _assertMorphoCap(Ethereum.MORPHO_VAULT_DAI_1, susde77,    0);
+        _assertMorphoCap(Ethereum.MORPHO_VAULT_DAI_1, ptsusdeoct, 0);
+        _assertMorphoCap(Ethereum.MORPHO_VAULT_DAI_1, ptsusdedec, 0);
+        _assertMorphoCap(Ethereum.MORPHO_VAULT_DAI_1, ptsusdemar, 400_000_000e18, 500_000_000e18);
+        _assertMorphoCap(Ethereum.MORPHO_VAULT_DAI_1, ptsusdemay, 0,              200_000_000e18);
+
+        skip(1 days);
+        IMetaMorpho(Ethereum.MORPHO_VAULT_DAI_1).acceptCap(ptsusdemar);
+        IMetaMorpho(Ethereum.MORPHO_VAULT_DAI_1).acceptCap(ptsusdemay);
+        
+        _assertMorphoCap(Ethereum.MORPHO_VAULT_DAI_1, ptsusdemar, 500_000_000e18);
+        _assertMorphoCap(Ethereum.MORPHO_VAULT_DAI_1, ptsusdemay, 200_000_000e18);
+    }
+
+    function test_ETHEREUM_StablecoinUpdates() public {
+        ReserveConfig[] memory allConfigsBefore = createConfigurationSnapshot('', pool);
+
+        ReserveConfig memory daiConfigBefore  = _findReserveConfigBySymbol(allConfigsBefore, 'DAI');
+        ReserveConfig memory usdcConfigBefore = _findReserveConfigBySymbol(allConfigsBefore, 'USDC');
+        ReserveConfig memory usdtConfigBefore = _findReserveConfigBySymbol(allConfigsBefore, 'USDT');
+
+        assertEq(daiConfigBefore.interestRateStrategy, OLD_DAI_INTEREST_RATE_STRATEGY);
+        assertEq(usdcConfigBefore.interestRateStrategy, OLD_STABLECOINS_INTEREST_RATE_STRATEGY);
+        assertEq(usdtConfigBefore.interestRateStrategy, OLD_STABLECOINS_INTEREST_RATE_STRATEGY);
+
+        IDefaultInterestRateStrategy prevIRM = IDefaultInterestRateStrategy(usdcConfigBefore.interestRateStrategy);
+        uint256 currVarSlope1 = 0.11885440509995120663752e27;
+        _validateInterestRateStrategy(
+            address(prevIRM),
+            OLD_STABLECOINS_INTEREST_RATE_STRATEGY,
+            InterestStrategyValues({
+                addressesProvider:             address(prevIRM.ADDRESSES_PROVIDER()),
+                optimalUsageRatio:             prevIRM.OPTIMAL_USAGE_RATIO(),
+                optimalStableToTotalDebtRatio: prevIRM.OPTIMAL_STABLE_TO_TOTAL_DEBT_RATIO(),
+                baseStableBorrowRate:          currVarSlope1,
+                stableRateSlope1:              prevIRM.getStableRateSlope1(),
+                stableRateSlope2:              prevIRM.getStableRateSlope2(),
+                baseVariableBorrowRate:        prevIRM.getBaseVariableBorrowRate(),
+                variableRateSlope1:            currVarSlope1,
+                variableRateSlope2:            prevIRM.getVariableRateSlope2()
+            })
+        );
```

Comment:
Should add assertion for DAI IRM here as well

Reply:
added